src/AWS.Deploy.DockerEngine/Templates/Dockerfile.Net6.template

```diff
@@ -0,0 +1,26 @@
+FROM {docker-base-image} AS base{non-root-user}
```

Comment:
The fix to container deployments only applies to .NET 7 and newer, so this dockerfile template is the version we currently use and will continue to be used for .NET6 and older targets.